docs/Change_Log.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## Premium Interactions
+
+#### May 16, 2024
+
+**Premium Apps: New Premium Style for MessageComponentType.BUTTON**
+
+This button is to be used with `sku_id` which points to an active SKU. This allows developers to customize their premium experience.
```

Comment:
```suggestion
Introducing a new `premium` [button style](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/button-object-button-styles) to be used with an `sku_id` which points to an active [SKU](#DOC_MONETIZATION_SKUS/sku-object). This allows developers to customize their premium experience.
```

Reply:
I can commit this but we need to change `an` -> `a`.